test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs

```diff
@@ -5513,6 +5520,30 @@ public void PackCommand_PackageReadmeFile_BasicFunc_WithSymbol_Succeeds()
             }
         }
 
+        [PlatformFact(Platform.Windows)]
+        public void PackCommand_PackageReadmeFile_MissingReadmeFile_ShowInfo()
+        {
+            var testDirBuilder = TestDirectoryBuilder.Create();
+            var projectBuilder = ProjectFileBuilder.Create();
+
+            testDirBuilder
+                .WithFile("test\\readme.md", 0);
+
+            projectBuilder
+                .WithProjectName("test")
+                .WithProperty("IncludeSymbols", "true")
+                .WithProperty("SymbolPackageFormat", "snupkg");
+
+            using (var srcDir = msbuildFixture.Build(testDirBuilder))
+            {
+                projectBuilder.Build(msbuildFixture, srcDir.Path);
+                var result = msbuildFixture.PackProject(projectBuilder.ProjectFolder, projectBuilder.ProjectName, string.Empty, validateSuccess: false);
```

Comment:
Considering there a "unit" test in `NuGet.Packagaing.Test`, does this test prevent any other regression? Tests like this that run a process, wait for the process the finish, and then assert the outcome as significantly slower than tests that run code in the test host process. Plus, duplicated tests make CI run slower without providing benefit.

Reply:
Thanks! I removed the two functional tests.